ssas/service/main/main.go

```diff
@@ -190,8 +190,7 @@ func newForwardingRouter() http.Handler {
 }
 
 func addFixtureData() {
-	db := ssas.GetGORMDbConnection()
-	defer ssas.Close(db)
+	db := ssas.Connection
```

Comment:
We could probably inline `ssas.Connection` in all of these tests instead of assigning a new variable, but I wanted to keep the changes in this PR as minimal as possible for clarity. I'll follow up afterwards with some test cleanup.

Reply:
Appreciate keeping the PR small since this update touched a lot of files. Will keep that in mind for the next PR! 👍